app.json

```diff
@@ -48,8 +58,9 @@
         "origin": false
       },
       "eas": {
-        "projectId": "adf55b61-1d80-4a97-9a42-b626cc03d958"
+        "projectId": "5ec73cc6-61fc-4ef7-abee-e218fa01fc35"
```

Comment:
about this, i changed accidentally, please fixed for ur projectId: adf55b61-1d80-4a97-9a42-b626cc03d958 @tomasvainstein  @ijolodovsky